File: openfinalfinal/u202120418/u202120418/src/main/java/org/hign/platform/u202120418/assessment/domain/model/commands/CreateMentalStateExamCommand.java
package org.hign.platform.u202120418.assessment.domain.model.commands;

/**
 * @author Fiorella Jarama Peñaloza
 * @version 1.0
 */
public record CreateMentalStateExamCommand(
        Long patientId,
        String examDate,
        Integer orientationScore,
        Integer attentionAndCalculationScore,
        Integer registrationScore,
        Integer recallScore,
        Integer languageScore,
        String examinerNationalProviderIdentifier
) {
}

File: web/HIGN/HIGN/Assessment/Domain/Model/Commands/CreateMentalStateExamCommand.cs
namespace HIGN.Assessment.Domain.Model.Commands;

public record CreateMentalStateExamCommand(long PatientId, string NationalProvideIdentifier, DateTime ExamDate, int OrientationScore, int RegistrationScore, int AttentionAndCalculationScore, int RecallScore, int LanguageScore, long ExaminerId);